tests/conftest.py

```diff
@@ -1492,6 +1492,104 @@ def titanic_pandas_data_context_with_v013_datasource_stats_enabled_with_checkpoi
     return context
 
 
+@pytest.fixture
```

Comment:
We might embellish this fixture later (e.g., if additional Fluent Datasources and/or DataAssets are needed).